port_ocean/clients/port/mixins/blueprints.py

```diff
@@ -14,13 +14,15 @@ def __init__(self, auth: PortAuthentication, client: httpx.AsyncClient):
         self.auth = auth
         self.client = client
 
-    async def get_blueprint(self, identifier: str) -> Blueprint:
+    async def get_blueprint(
+        self, identifier: str, should_log: bool = True
+    ) -> Blueprint:
         logger.info(f"Fetching blueprint with id: {identifier}")
         response = await self.client.get(
             f"{self.auth.api_url}/blueprints/{identifier}",
             headers=await self.auth.headers(),
         )
-        handle_status_code(response)
+        handle_status_code(response, should_log=should_log)
```

Comment:
Why is `should log` added only here? should it not be in other places? is it necessary for it to be on get_blueprint specifically?

Reply:
Note line https://github.com/port-labs/ocean/pull/308/files/ffaf3b7ef7fa079d5ff99707d45275626d18e1e9#diff-805c13724e2962cfe2a15019fe6cbfb8dcd894d15cda5d0edb414dbbf84fd64cR70 which eventually goes over all blueprints which we need to create and checks if its exists, we want the resource to not exists which in this case will raise an exception, but the exception is actually a good thing so I don't want to log an error in this case.